src/workflow-design-guide/style-guide.rst

```diff
@@ -92,45 +92,49 @@ to, and trailing comments should be preceded by two spaces, as shown above.
 Script String Lines
 ^^^^^^^^^^^^^^^^^^^
 
-Script strings are written verbatim to task job scripts so they should really
-be indented from the left margin:
+Script strings are written verbatim to task job scripts.
+
+.. code-block:: cylc
+
+   [runtime]
+       [[foo]]
+           script = echo "Hello, Mr. Thompson"
+
+If using a triple-quoted string, any common leading whitespace is trimmed
+using the logic of
+`Python's textwrap.dedent <https://docs.python.org/2/library/textwrap.html#textwrap.dedent>`_.
```

Comment:
We have hooked up the Python docs via intersphinx so you can use something like:

```rst
:py:func:`textwrap.dedent`
```

To reference `dedent` in the Python 3 docs.